pkg/cassandra/session.go

```
@@ -181,6 +181,22 @@ func (s *Session) Close() {
 	s.session.Close()
 }
 
+// firstResponsiveHost returns whichever contact point answers first, so that
+// opening a session is not held up by a replica that is slow or down.
+func firstResponsiveHost(hosts []string, ping func(string) error) string {
```

golangci-lint (2.12.2)

[error] func firstResponsiveHost is unused

(unused)